daml-lf/spec/daml-lf-1.rst

```diff
@@ -413,14 +413,7 @@ and other similar pitfalls. ::
      PackageIdChar  ∈  [a-zA-Z0-9\-_ ]               -- PackageIdChar
 
   PartyId strings
-     PartyIdString ::= ' PartyIdChars '               -- PartyIdString
-
-  Sequences of PartyId character
```

Comment:
I always got confused why this wasn’t just a regex before. Thanks for cleaning it up!